### DISCUSSION TITLE
Why no scrollbar?

### DISCUSSION THREAD

**Discussion by Free-Radical** [kovidgoyal/kitty#6358] [Q&A]
Just switched to this hidden gem 👏. 
Been googling and find there is no scoll bar.
Coming from other terms I'm super used to having this option, am i missing how to turn it on? or there is no scrollbar, if so, I was wondering why?

**Comment by otakutyrant** [comment]
Read [this](https://github.com/kovidgoyal/kitty/issues/2502#issuecomment-606996263).

**Reply by cD1rtX3** [reply]
You mean [this](https://www.reddit.com/r/programmingcirclejerk/comments/14lhw1f/im_afraid_i_have_no_interest_in_implementing_a/)? Lol. Lmao, even.

Even in the linked issue, the developer is getting clowned. Does this man just… not take criticism? Does he think it's a personal attack or something?

**Comment by zzhaolei** [comment]
It is now supported and enabled by default.

**Reply by cD1rtX3** [reply]
As of 0.43.0, yes. Thank you for notifying me! Ubuntu is still stuck on 0.32.2, but that's because Ubuntu doesn't like updating.
